server/vulnerabilities/nvd/cpe.go

```diff
@@ -336,14 +386,26 @@ func TranslateSoftwareToCPE(
 			level.Error(logger).Log("software->cpe", "error translating to CPE, skipping...", "err", err)
 			continue
 		}
-		if cpe == "" {
+		if cpe == software.GenerateCPE {
```

Comment:
Q: What if `software.GenerateCPE` and cpe are both `""`? (Guessing that scenario means a CPE wasn't found previously and is still not found?, should we log that?)

Reply:
Yes, that means that we are not able to determine the CPE or that there's a ignore rule for it. We could add a logging statement for that scenario but that could get a little noisy, plus there won't too much value to it in terms of debugging IMHO.